tests/component/system/storage/test_persisted_channel.py

```diff
@@ -29,3 +41,23 @@ def test___persisted_channels_with_different_names___hash___not_equal(init_kwarg
     persisted_channel2 = PersistedChannel("Channel2", **init_kwargs)
 
     assert hash(persisted_channel1) != hash(persisted_channel2)
+
+
+def test___save_persisted_channel___no_error(ai_task, init_kwargs):
```

Comment:
Please use type hints in new tests. We don't have to be overly precise about init_kwargs, though.

```suggestion
def test___save_persisted_channel___no_error(ai_task: Task, init_kwargs: dict) -> None:
```

Reply:
Changed.